extensions/hibernate-search-elasticsearch/runtime/src/main/java/io/quarkus/hibernate/search/elasticsearch/runtime/HibernateSearchElasticsearchRuntimeConfig.java

```diff
@@ -63,7 +63,7 @@
          * The list of hosts of the Elasticsearch servers.
          */
         @ConfigItem(defaultValue = "http://localhost:9200")
-        Optional<List<String>> hosts;
+        List<String> hosts;
```

Comment:
It's normal there are no other code changes as we have a config framework in the Search extension that deals with either Optional or not.